### DISCUSSION TITLE
Distance Duality Test: The Evolution of Radio Sources Mimics a Nonexpanding Universe

### DISCUSSION THREAD

**Discussion by pierre-real** [a-cosmology-group/acg#83] [Cosmology Models]
Distance Duality Test: The Evolution of Radio Sources Mimics a Non expanding Universe
https://arxiv.org/pdf/2306.15680.pdf
The problem identified in this paper points two ways forward: (1) the Universe is not expanding, so we would need to re-explain the observed cosmological redshift and other related observations; (2) the Universe is expanding, so we would have to explain why the evolution of ultra compact radio sources precisely mimics a non expanding Universe.

**Comment by budrap00** [comment]
Good one Pierre! This strikes at the expanding universe disease rather than the symptoms thereof like dark matter etc. Funny that the author sort of feints toward the second, orthodox choice but then throws in a non-careerist tilt toward science:

> Weighing the price, option (2) is obviously more attractive. However, science is not about bargaining a better deal, but finding out the truth.

**Reply by RedshiftDrift** [reply]
Thanks Pierre, this is a *very* good one!  
The published version is `Open Access`, available at [https://dx.doi.org/10.3847/2041-8213/acdb49](https://dx.doi.org/10.3847/2041-8213/acdb49)

> the author sort of feints toward the second, orthodox choice...

Required if he wants to be published.

> but then throws in a non-careerist tilt toward science...

Pengfei Li is an Alexander von Humboldt fellow, a post doc with almost total freedom of research.  He is the only author on the paper (perhaps too risky for his supervisor) but Stacy McGaugh was on his PhD thesis committee.  That may explain the *non-careerist tilt*.

What surprises me (in a good way) is that he parameterises the linear size of radio sources as

$$d = d_0 (1+z)^a \ [\ln(1+z)]^b \mbox{\hspace{20pt} Eq. (4)}$$
which has the same form as the general equation used in tired-light models 😉

**Reply by pierre-real** [reply]
A search for "Pengfei Li" on arXiv results in 113 papers. Of them only two are from himself only, all others are with many other collaborators. 

As can be deduced from a quick look at all those papers one can say that Pengfei Li in a person who can apprehend any complicated problems in any field of interest to create a realistic understanding of it. So it is a person of interest for other researchers in any field explaining such presence in 111 papers .

This being and with Redshift Drift 's comment giving a possible reason for the fact that Pengfei Li is the sole author of this paper brought my attention on it.  So I searched for other papers for which he was the sole author and I found this second one also tackling a cosmological problem:  https://browse.arxiv.org/pdf/2305.19289.pdf  

Of interest is his last sentence in section 4 before the conclusion:
"Accommodating these super cuspy halos poses a challange for the CDM model."

So challenging the cosmological dogma single you out.

**Reply by RedshiftDrift** [reply]
Thanks @pierre-real 
The [cuspy halo problem](https://en.wikipedia.org/wiki/Cuspy_halo_problem) is a known challenge to CDM, and papers often discuss it.  Yet [some astronomers](https://www.forbes.com/sites/startswithabang/2019/10/18/dark-matters-biggest-problem-might-simply-be-a-numerical-error/?sh=218932c38979) are still in denal.

Of interest is the sentence in the conclusion *"For massive galaxies, one might need to introduce more powerful feedback processes."*  Tired-light redshift implies a "re-[deposition of energy and momentum into the interstellar medium of a galaxy through radiation](https://www2.ccs.tsukuba.ac.jp/Astro/Members/ayw/research/agn_feedback/agn_feedback.html)", just what is needed for feedback.

Pengfei Li could become an important ally for me.

**Reply by bgreenman** [reply]
Pengfei Li is in a fortunate situation, in that he able to publish his on view on tired-light redshift theory,because he i an alexander von humboldt fellow with full fredom to pursue the truth
